easybuild/easyblocks/n/nwchem.py

```diff
@@ -500,6 +508,8 @@ def test_cases_step(self):
                 # run tests
                 for testx in tests:
                     cmd = "nwchem %s" % testx
+                    if self.cfg['mpi_exe']:
```

Comment:
If I remember correctly this is already defined in the respective MPI framework? @boegel ??